content/docs/tutorials/interactivity/remote-data@es.md

```diff
@@ -0,0 +1,182 @@
+        "color": "black",
+        "image": "./shirts/black.jpg",
+        "sizes": {
+          "XS": 8.99,
+          "S": 9.99
+        }
+      },
+<!-- ... -->
+```
+
+Y, debemos actualizar el estado predeterminado de los elementos relevantes:
+
+```html
+<amp-selector name="size">
+  <table>
+    <tr>
+      <!-- If 'XS' size is available for selected SKU, return empty string.
+           Otherwise, return 'unavailable'. -->
+      <td [class]="shirts[selected.sku].sizes['XS'] ? '' : 'unavailable'">
+        <div option="XS">XS</div>
+      </td>
+      <td [class]="shirts[selected.sku].sizes['S'] ? '' : 'unavailable'">
+        <div option="S">S</div>
+      </td>
+      <!-- Add the 'unavailable' class to the next three <td> elements
+           to be consistent with the available sizes of the default SKU. -->
+      <td class="unavailable" 
+          [class]="shirts[selected.sku].sizes['M'] ? '' : 'unavailable'">
+        <div option="M">M</div>
+      </td>
+      <td class="unavailable" 
+          [class]="shirts[selected.sku].sizes['L'] ? '' : 'unavailable'">
+        <div option="L">L</div>
+      </td>
+      <td class="unavailable" 
+          [class]="shirts[selected.sku].sizes['XL'] ? '' : 'unavailable'">
+        <div option="XL">XL</div>
+      </td>
+    </tr>
+  </table>
+</amp-selector>
+```
+
+{% call callout('Nota', type='note') %}
+`<amp-bind>` no se ejecuta en carga de página, -- solo en respuesta a una acción explícita del usuario. Esto asegura que la carga inicial de la página sea consistentemente rápida entre las páginas independientemente del uso de `<amp-bind>`.
+{% endcall %}
+
+## Precios variables
+
+Ahora que mostramos correctamente los tamaños disponibles, nos aseguramos de que el precio correcto también se muestre.
+
+Nuestra tienda de AMPPAREL es peculiar en que el precio de la camisa es específico para el color Y el tamaño. Esto significa que necesitamos una nueva variable para rastrear el tamaño seleccionado por el usuario. Agregue una nueva acción a nuestro elemento `<amp-selector>`:
+
+```html
+<!-- When an element is selected, set the `selectedSize` variable to the
+     value of the "option" attribute of the selected element.  -->
+<amp-selector name="size" 
+    on="select:AMP.setState({selectedSize: event.targetOption})">
+```
+
+Tenga en cuenta que no estamos inicializando el valor de `selectedSize` a través del elemento `amp-state#selected`. Eso es porque intencionalmente no proporcionamos un tamaño seleccionado por defecto y en lugar de eso queremos obligar al usuario a elegir un tamaño.
+
+{% call callout('Tip', type='success') %}
+`AMP.setState()` se puede utilizar para definir nuevas variables además de modificar las existentes. Las expresiones evaluarán las variables indefinidas a `null`.
+{% endcall %}
+
+Añada un nuevo elemento `<span>` que enrolle la etiqueta de precio y cambie el texto predeterminado a "---" ya que no hay una selección de tamaño predeterminada.
+
+```html
+<h6>PRECIO :
```

Comment:
Please change all the code back to english, for consistency and to avoid confusion.

Reply:
Done